test/prism_regression/array.parse-tree.exp

```diff
@@ -105,7 +105,7 @@ Begin {
       args = Args {
         args = [
           Restarg {
-            name = <P <U *> $2>
+            name = <U *>
```

Comment:
Yep, I think this is an expected change from https://github.com/sorbet/sorbet/pull/8203/files#diff-fe831845ab9cf399bcfbbe7cfb30176cb8ff1b5fd6c23f61a37c8909c127f5bb